tests/template_tests/test_autoreloader.py

```diff
@@ -32,6 +33,17 @@
     ],
 )
 class TemplateReloadTests(SimpleTestCase):
+    @override_settings(FORM_RENDERER="django.forms.renderers.TemplatesSetting")
+    @mock.patch("django.template.loaders.cached.Loader.reset")
+    def test_form_template_reset_template_change_no_djangotemplates(
+        self, mock_loader_reset
+    ):
+        get_default_renderer.cache_clear()
+        template_path = Path(__file__).parent / "templates" / "index.html"
+        self.assertIs(autoreload.template_changed(None, template_path), True)
+        mock_loader_reset.assert_not_called()
+        get_default_renderer.cache_clear()
```

Comment:
If this `cache_clear` call is needed at the end to avoid test interference/test contamination, I would suggest to do instead:

```suggestion
        self.addCleanup(get_default_renderer.cache_clear)
```

Also please note that this test would be best placed right after `test_form_template_reset_template_change_reset_call` as suggested in the ticket.